core/src/integrationTest/kotlin/ch/ergon/dope/helper/BaseIntegrationTest.kt

```diff
@@ -0,0 +1,71 @@
+package ch.ergon.dope.helper
```

Comment:
not really a helper-class - Would rename the package to _integrationtest_ or something

Reply:
done